minecraft-headers/minecraft/ItemRegistry.h

```diff
@@ -17,6 +20,8 @@ struct ItemRegistry{
 	static std::unordered_map<HashedString, ItemAlias> mItemAliasLookupMap;
 	static std::unordered_map<HashedString, std::function<HashedString(short)>> mComplexAliasLookupMap;
 
-	static std::pair<HashedString, int> getNameFromAlias(HashedString const & name, int metadata);
+    static std::unordered_map<ItemTag, std::unordered_set<Item const *, std::hash<Item const *>, std::equal_to<Item const *>, std::allocator<Item const *> >, std::hash<ItemTag>, std::equal_to<ItemTag>, std::allocator<std::pair<ItemTag const, std::unordered_set<Item const *, std::hash<Item const *>, std::equal_to<Item const *>, std::allocator<Item const *> > > > > mTagToItemsMap;
```

Comment:
I think you can shorten the `unordered_set` declaration. Apart from the first template parameter, the rest of the parameters can be automatically inferred and don't need to be written in the code like this.

Reply:
```suggestion
    static std::unordered_map<ItemTag, std::unordered_set<Item const *>> mTagToItemsMap;
```